bin/scalafmt.ps1

```diff
@@ -0,0 +1 @@
+&java -jar "$pwd\bin\scalafmt" $args
```

Comment:
Since we are running stryker4s on a linux server it would make more to make a bash script

Reply:
The file it calls is also a Linux shell script. This is just for Windows compatibility